ntp/syncTime.go

```diff
@@ -220,6 +228,26 @@ func (s *syncTime) sync() {
 		"clock offset harmonic mean", clockOffsetHarmonicMean)
 }
 
+func (s *syncTime) isResponseTimeWithinAcceptedBounds(
+	responseDurations int64,
+	numSuccessfulRequests int64,
+) bool {
+	if numSuccessfulRequests <= 0 {
+		return false
```

Comment:
is this correct here? or should be return true? 🤔

Reply:
if no successful requests we consider response to be out of bounds, but in the end, the behaviour will be the same as before, due to the other conditions

Reply:
add one more unit test case